models/user.js

```diff
@@ -40,8 +40,8 @@ module.exports.getUserById = function(id, callback){
   User.findById(id, callback);
 }
 
-module.exports.getSettingsById = function(id, callback){
-  User.findById(id, callback).select('settings -_id');
+module.exports.getNotificationInfoById = function(id, callback){
```

Comment:
change name on line 111 in index.js as well

Reply:
there is no need to do this, that api is fine

It calls getSettingsByUserId,

the one I changed is different